lab/lab1/part1.js

```diff
@@ -82,39 +82,39 @@ underscore. Should evaluate to true.
 ===================== */
 
 var query3;
-
+query3 = _isObject(_.first(bakedGoods));
 console.log('The first element in bakedGoods is an object:', query3);
 
 /* =====================
 Use _.where to return all cakes. Or bread. Whichever is your favorite.
 ===================== */
 
 var query4;
-
+query4 = _.where(bakedGoods, {type:'Cake'});
 console.log('All bread. Or cakes:', query4);
 
 /* =====================
 Use _.filter to return all baked goods that cost more than $4.
 ===================== */
 
 var query5;
-
+query5 = ._filter(bakedGoods, function(obj) {return obj.price>4;});
```

Comment:
order is important with the `.`. and `_`

Reply:
Changed